types/intl-tel-input/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for intl-tel-input 17.0
+// Type definitions for intl-tel-input 17.0.3
```

Comment:
```suggestion
// Type definitions for intl-tel-input 17.0
```
There is no patchin in DT SemVer:
https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-definitely-typed-package-versions-relate-to-versions-of-the-corresponding-library